hashtagmap-admin/src/main/java/com/songpapeople/hashtagmap/instagram/service/InstagramCommandService.java

```diff
@@ -0,0 +1,35 @@
+package com.songpapeople.hashtagmap.instagram.service;
+
+import com.songpapeople.hashtagmap.blacklist.service.dto.BlackListRequest;
+import com.songpapeople.hashtagmap.exception.AdminException;
+import com.songpapeople.hashtagmap.exception.CommonExceptionStatus;
+import com.songpapeople.hashtagmap.instagram.domain.model.Instagram;
+import com.songpapeople.hashtagmap.instagram.domain.repository.InstagramRepository;
+import com.songpapeople.hashtagmap.place.domain.model.Place;
+import com.songpapeople.hashtagmap.place.domain.repository.PlaceRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class InstagramCommandService {
+    private final InstagramRepository instagramRepository;
+    private final PlaceRepository placeRepository;
+
+    public InstagramCommandService(InstagramRepository instagramRepository, PlaceRepository placeRepository) {
+        this.instagramRepository = instagramRepository;
+        this.placeRepository = placeRepository;
+    }
+
+    @Transactional
+    public void delete(BlackListRequest blackListRequest) {
+        instagramRepository.delete(findInstagramByPlaceId(blackListRequest.getPlaceId()));
+    }
+
+    public Instagram findInstagramByPlaceId(Long placeId) {
+        Place place = placeRepository.findById(placeId)
+                .orElseThrow(() -> new AdminException(
+                        CommonExceptionStatus.NOT_PERSIST,
+                        String.format("존재하지 않는 placeId(%d)입니다.", placeId)));
+        return instagramRepository.findByPlaceFetch(place);
+    }
```

Comment:
```suggestion
    @Transactional
    public void delete(BlackListRequest blackListRequest) {
        Place place = placeRepository.findById(blackListRequest.getPlaceId())
                .orElseThrow(() -> new AdminException(
                        CommonExceptionStatus.NOT_PERSIST,
                        String.format("존재하지 않는 placeId(%d)입니다.", blackListRequest.getPlaceId())));
        Instagram instagram = instagramRepository.findByPlaceFetch(place);

        instagramRepository.delete(instagram);
    }
```

굳이 메서드로 빼지 않아도 충분히 이해가 가는 부분이라 메서드로 추출하지 않아도 될것 같아요!
재사용성을 위해 뺀다면 나중에 재사용하는 부분이 생겼을때 빼는게 맞을것 같아요!
그리고 정 빼야 한다면 public이 아닌 private 메서드가 되어야 할 것 같아요!